eventdiscovery-sdk/src/main/res/layout/schedjoules_details_header.xml

```diff
@@ -16,8 +16,8 @@
                 android:fitsSystemWindows="true"
                 app:expandedTitleGravity="bottom|left"
                 app:expandedTitleTextAppearance="@style/SchedJoules_ExpandedTitle"
-                app:expandedTitleMarginStart="@dimen/schedjoules_activity_horizontal_margin"
-                app:expandedTitleMarginEnd="@dimen/schedjoules_activity_horizontal_margin"
+                app:expandedTitleMarginStart="17dp"
```

Comment:
Sorry, I didn't express it well, I meant that it's get aligned correctly with the other texts when its margin is set to 17dp.